src/base/bittorrent/session.cpp

```diff
@@ -2982,6 +3020,14 @@ bool Session::hasPerTorrentRatioLimit() const
     return false;
 }
 
+bool Session::hasPerTorrentSeedingTimeLimit() const
+{
+    foreach (TorrentHandle *const torrent, m_torrents)
+        if (torrent->seedingTimeLimit() >= 0) return true;
+
```

Comment:
Remember, everything I have done is because somewhere else in the same file is like that and it would look inconsistent.  The complete section is like this, only the second method is my code:
```
bool Session::hasPerTorrentRatioLimit() const
{
    foreach (TorrentHandle *const torrent, m_torrents)
        if (torrent->ratioLimit() >= 0) return true;

    return false;
}

bool Session::hasPerTorrentSeedingTimeLimit() const
{
    foreach (TorrentHandle *const torrent, m_torrents)
        if (torrent->seedingTimeLimit() >= 0) return true;

    return false;
}
```
And I think it looks better that way, like the original coder ;)